### PR TITLE
Label CRUD

### PR DESCRIPTION
**Pull request by BadBrahmin**
Pr for Issue #3

### CODE REVIEW

File: feature/controllers/labelController.js [hunk 1]
```diff
@@ -14,4 +28,93 @@ module.exports = {
...
+      });
+  },
+
+  updateLabel: (req, res) => {
+    const id = req.params.id;
+    // const issues = [];
+
+    const { name, description, color, issues } = req.body;
+    if (!name || !description || !color) {
+      return res.json({
```
Comment (harshaanNihal) [inline feature/controllers/labelController.js]: An appropriate status code should be present in responses.

File: feature/app.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const logger = require('morgan');
 const mongoose = require('mongoose');
 
 const indexRouter = require('./routes/index');
-// const usersRouter = require('./routes/users');
```
Comment (harshaanNihal) [inline feature/app.js]: Unwanted comments should not be present in PR.

Comment (BadBrahmin) [reply feature/app.js]: Has been removed in the last commit.

File: feature/controllers/labelController.js (hunk 1 above)
Comment (BadBrahmin) [reply feature/controllers/labelController.js]: added in commit 029bc9b

**Pull request merged**